src/transforms/metric_to_log.rs

```diff
@@ -63,6 +63,17 @@ pub struct MetricToLogConfig {
     pub metric_tag_values: MetricTagValues,
 }
 
+impl MetricToLogConfig {
+    pub fn build(&self, context: &TransformContext) -> MetricToLog {
```

Comment:
I'm not excited about the overloaded `fn build` name here. I was confused about no longer needing the `.await` above and couldn't figure out how that worked until I saw this. Maybe call it `build_inner` or `build_sync` instead? I'm not hung up on what it's called, I just don't like the duplication.

Reply:
That is a very reasonable concern.